src/pages/index.js

```diff
@@ -70,31 +155,38 @@ function createCard(data) {
       handleImageClick: () => {
         imageModal.open(data);
       },
+      handleLikeClick: (card) => {
+        api
+          .likeCardStatus(card.getId(), !card._isLiked)
+          .then((data) => {
+            card.handleLike(data.isLiked);
+          })
+          .catch((err) => {
+            console.log(err);
+          });
+      },
+      handleDeleteCardClick: confirmDeleteCard,
     },
     cardSelectors.cardTemplate,
   );
   return card.getView();
 }
 
-// /* Event Handlers */
-
-function handleProfileEditFormSubmit(profileData) {
-  const name = profileData.title;
-  const bio = profileData.bio;
-  user.setUserInfo({ name, bio });
-  editProfileModal.close();
+function confirmDeleteCard(cardData) {
+  confirmDeleteModal.open();
+  confirmDeleteModal.confirmDelete(() => {
+    api
+      .deleteCard(cardData.getId())
+      .then(() => {
+        cardData.handleDeleteCard();
+        confirmDeleteModal.close();
+      })
+      .catch((err) => {
+        console.log(err);
+      });
+  });
 }
-
-function handleAddCardFormSubmit(newCardData) {
-  console.log(newCardData);
-  const name = newCardData.title;
-  const alt = newCardData.title;
-  const link = newCardData.link;
-  section.addItem(createCard({ name, alt, link }));
-  newCardModal.close();
-}
-
-// /* Event Listeners */
+/* Event Listeners */
 
 profileEditButton.addEventListener("click", () => {
   const userInput = user.getUserInfo();
```

Comment:
Please, call ` editFormValidator.resetValidation();` for the profile opening to clear possible errors from the inputs